packages/transformer/src/test/standalone/IModelExporter.test.ts

```diff
@@ -73,4 +73,36 @@ describe("IModelExporter", () => {
 
     sourceDb.close();
   });
+
+  describe("exportRelationships", () => {
+    it("should not export relationships that do not have source or target elements", async () => {
+      const sourceDbPath = IModelTransformerTestUtils.prepareOutputFile("IModelExporter", "InvalidRelationship.bim");
+      const sourceDb = SnapshotDb.createEmpty(sourceDbPath, { rootSubject: { name: "invalid-relationships" } });
+
+      const relationshipProps: RelationshipProps = {
```

Comment:
can you add to this test:
1. a relationship with valid ids, where you insert two elements, create a relationship between them, and then delete one of the elements?
2. a relationship with valid source, invalid target
3. a relationship with invalid source, valid target

(it would be nice btw if you posted that message you sent to me as the PR description). If I understand the source problem you sent to me correctly, you said the "other element is non existent" in the invalid relationships you're seeing, which means I think we need to test that directly, aka case 1 above.

Ideally I would actually like us to transform the invalid relationships in case 1 but nullify the id they point to, based on whether the transformer options' `danglingPredecessorsBehavior` is set to reject or not... thoughts?

Reply:
to add to this, if you personally want your transformer implementation to ignore such "invalid" relationships, you can do:

```ts
public override shouldExportRelationship(sourceRel) {
  return Id64.isValid(sourceRel.sourceId) && Id64.isValid(sourceRel.targetId);
}
```

Reply:
Since it doesn't work anyway at the moment, I have put up #21 to fix this in the future. I will keep the comment open because the list of test cases I would like you to add still stand.